From b7a24467f738bfd9437e8de02ed5e44409054185 Mon Sep 17 00:00:00 2001
From: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
Date: Wed, 23 Jul 2014 11:53:33 -0700
Subject: [PATCH 3/4] ListenOnOpenFD: Remove Resets since this is intended to
 be for hotplugging connections

pharris says that the resets should not be done in the hotplugging case.

This may fix a crash reported against XQuartz:
http://xquartz.macosforge.org/trac/ticket/869

Signed-off-by: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
Reviewed-by: Peter Harris <pharris@opentext.com>
(cherry picked from commit dfbc6a1a78ff3192baae6f7f2af98b6e51422f75)
---
 os/connection.c | 7 -------
 1 file changed, 7 deletions(-)

diff --git xorg-server-1.16.0/os/connection.c xorg-server-1.16.0/os/connection.c
index 40d9ff3..f915e12 100644
--- xorg-server-1.16.0/os/connection.c
+++ xorg-server-1.16.0/os/connection.c
@@ -1307,13 +1307,6 @@ ListenOnOpenFD(int fd, int noxauth)
 
     /* Increment the count */
     ListenTransCount++;
-
-    /* This *might* not be needed... /shrug */
-    ResetAuthorization();
-    ResetHosts(display);
-#ifdef XDMCP
-    XdmcpReset();
-#endif
 }
 
 /* based on TRANS(SocketUNIXAccept) (XtransConnInfo ciptr, int *status) */
-- 
2.0.4

